modules/default-values/assets/js/editor/component.js

```diff
@@ -0,0 +1,33 @@
+import * as commands from './commands/index';
+import * as dataCommands from './commands-data/index';
+import GlobalValues from './handlers/global-values';
+import LocalValues from './handlers/local-values';
+
+export default class Component extends $e.modules.ComponentBase {
+	getNamespace() {
+		return 'default-values';
+	}
+
+	defaultCommands() {
+		return this.importCommands( commands );
+	}
+
+	defaultData() {
+		return this.importCommands( dataCommands );
+	}
+
+	__construct( args = {} ) {
```

Comment:
Move up

Reply:
Try use `registerAPI` instead.
```suggestion
	registerAPI() {
```